src/Symfony/Component/Security/Http/Tests/Firewall/RememberMeListenerTest.php

```diff
@@ -248,10 +313,11 @@ protected function getListener($withDispatcher = false, $catchExceptions = true)
             $manager = $this->getManager(),
             $logger = $this->getLogger(),
             $dispatcher = ($withDispatcher ? $this->getDispatcher() : null),
-            $catchExceptions
+            $catchExceptions,
+            $sessionStrategy = $this->getSessionStrategy()
```

Comment:
I don't think we need to modify this test. Keeping it as is also ensures that the `RememberMeListener` still works when it gets no session strategy passed.